<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssociationEditDialog</class>
 <widget class="QDialog" name="AssociationEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="associationEnd1GroupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>521</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>AssociationEnd 1</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Role:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roleAEnd1LineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiplicity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeAEnd1ComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="multiplicityAEnd1LineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="associationNameEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="associationEnd2GroupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>521</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>AssociationEnd 2</string>
   </property>
   <widget class="QLineEdit" name="roleAEnd2LineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiplicity:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeAEnd2ComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Role:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="multiplicityAEnd2LineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Association type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="associationTypeComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AssociationEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AssociationEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
